rally.ts

```diff
@@ -5,12 +5,21 @@
 const CORE_ADDON_ID = "rally-core@mozilla.org";
 const SIGNUP_URL = "https://rally.mozilla.org/rally-required";
 
+import { browser, Runtime } from "webextension-polyfill-ts";
+
 export const runStates = {
 	RUNNING: "running",
 	PAUSED: "paused",
 }
 
 export class Rally {
+  private _namespace: any;
```

Comment:
it'd be good for us to add the type information on these. I thiiiink most of these are strings, right?

Reply:
Yeah,  I think that should be easy enough. For these I just let vscode guess :)

Reply:
OK I tightened up all of the types, no more `any`!

I'm pretty sure we can drop the existing key validation code, since all it does is check for an object with a `kid` (key ID) property, which the typescript compiler now checks for us. I think it would be a good idea to use a real JWK validator, I'll file an issue.